framework/src/wifi_manager/Kconfig

```diff
@@ -12,7 +12,7 @@ config WIFI_MANAGER
     ---help---
         Easy APIs for applications to use and control Wi-Fi features
 
-if WIFI_MANAGER
+if WIFI_MANAGER && ARCH_CHIP_S5JT200
```

Comment:
What is the purpose of this? Do you want not to use wifi on artik053?

Reply:
I expected to build supplicant only for s5jt200. But I understand that sourcing the Kconfig does not mean it gets built. So I will revert the changes. Thanks for pointing out.